vanilla/todo.js

```diff
@@ -14,22 +14,29 @@ const SingleTaskClassNames = {
 }
 
 const addTask = () => {
+    // maybe add a helper that wraps document.querySelector(xxx) instead of writing it every time?
     const form = document.querySelector('#new_task_form');
+    // you are only using task_input in this function why not select it directly?
     const input =  form.task_input.value;
 
     if (input != '') {
+        // why is this line here and not inside "createTask" ?
+        // It's nicer to write .xxx > .yyy
```

Comment:
wdym?

Reply:
that you can put `const tableBody = document.querySelector('.incomplete_tasks>tbody');` inside the `createTaskNewRow`